cas-server-core-logout/src/main/java/org/jasig/cas/logout/LogoutManagerImpl.java

```diff
@@ -68,6 +70,16 @@ public LogoutManagerImpl(final LogoutMessageCreator logoutMessageBuilder) {
             LOGGER.info("Single logout callbacks are disabled");
             return logoutRequests;
         }
+        performLogoutForTicket(ticket, logoutRequests);
+
+        final Collection<ProxyGrantingTicket> proxyGrantingTickets = ticket.getProxyGrantingTickets();
```

Comment:
Good call. I shall move this down. 

I am not sure what the future holds for logout management. @leleuj at some point was going to, and probably still is, working on front-channel SLO. So, keeping a separate abstraction to manage logout makes sense but at the same time, long way out, I am sure some of this stuff is due for a better rewrite.